resources/definitions/creality cr-x.def.json

```diff
@@ -0,0 +1,52 @@
+{
+    "id": "CR-X",
+    "version": 2,
+    "name": "Creality CR-X",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "SRC",
+        "manufacturer": "Creality",
```

Comment:
Please use "Creality3D" as the manufacturer so all the printers that belong to the same family uses the same manufacturer.